Examples/ImageGridManipulation/Documentation.rst

```diff
@@ -0,0 +1,60 @@
+Image Grid Manipulation
+=======================
+
+
+Overview
+--------
+
+There are numerous SimpleITK filters that have similar functions, but very
+important differences. The filters that will be compared are:
+
+ - ``JoinSeriesImageFilter()`` - Joins multiple N-D images into an (N+1)-D  image
+ - ``ComposeImageFilter()`` - Combines several scalar images into a  multicomponent vector image
+ - ``VectorIndexSelectionCastImageFilter()`` - Extracts the selected index of  the input pixel type vector (the input image pixel type must be a vector and the output a scalar)
```

Comment:
Please add something along the lines: Additionally, this filter can cast the output pixel type (``SetOutputPixelType`` method).

Explains the "Cast" in the filters name.